### PR TITLE
[IA-3152] Add one more PR site

### PR DESCRIPTION
**Pull request by dmohs**
Circle configuration updated to add NUM_PR_SITES as an environment variable. Usage of that variable is added in this PR. Any change to that variable would need a corresponding change in the web client authorized JavaScript origins:
https://console.cloud.google.com/apis/credentials/oauthclient/500025638838-s2v23ar3spugtd5t2v1vgfa2sp7ppg0d.apps.googleusercontent.com?project=bvdp-saturn-dev

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -38,7 +38,8 @@ commands:
                 command: |
                   apk add github-cli --no-progress --repository=https://dl-cdn.alpinelinux.org/alpine/latest-stable/community/
                   <<# parameters.pr >>
-                  VERSION="pr-$(expr "${CIRCLE_PR_NUMBER:-${CIRCLE_PULL_REQUEST##*/}}" % 10)"
+                  NUM_PR_SITES=11
```
Comment (dmohs) [inline .circleci/config.yml]:
```suggestion
```

File: .circleci/config.yml [hunk 2]
```diff
@@ -53,7 +54,8 @@ commands:
                 name: Update Github deployment status
                 command: |
                   <<# parameters.pr >>
-                  PR_SLUG="pr-$(expr "${CIRCLE_PR_NUMBER:-${CIRCLE_PULL_REQUEST##*/}}" % 10)-dot-"
+                  NUM_PR_SITES=11
```
Comment (dmohs) [inline .circleci/config.yml]:
```suggestion
```

File: .circleci/config.yml [hunk 3]
```diff
@@ -101,8 +103,9 @@ commands:
                 command: |
                   CIRCLE_PR_NUMBER="${CIRCLE_PR_NUMBER:-${CIRCLE_PULL_REQUEST##*/}}"
                   if [[ ! -z "$CIRCLE_PR_NUMBER" ]]; then
+                    NUM_PR_SITES=11
```
Comment (dmohs) [inline .circleci/config.yml]:
```suggestion
```

**Review by kyuksel — APPROVED** [review]
👍🏼 through mobbing

**Review by petesantos — APPROVED** [review]

**Review by s-rubenstein — APPROVED** [review]
LGTM

**Comment by zarsky-broad** [comment]
I personally would rather do something divisible by 10 to keep the number a bit more predictable, rather than like 11, but make sure you document where the number is defined.

You should take the changes to the file [from Christine's PR](https://github.com/DataBiosphere/terra-ui/pull/2740/files#diff-78a8a19706dbd2a4425dd72bdab0502ed7a2cef16365ab7030a5a0588927bf47).

**Comment by dmohs** [comment]
@zarsky-broad Agreed, but this keeps the change scope as small as possible. Made suggestions to the other PR.

**Pull request merged**